.travis.yml

```
@@ -33,4 +34,6 @@ jobs:
     - stage: deploy
       name: 'Deploy App'
       script:
-        - IMAGE=${GOOGLE_DOCKER_IMAGE}:${TRAVIS_BRANCH}-${TRAVIS_COMMIT:0:7} make deploy
+        - IMAGE=${IMAGE_BRANCH_COMMIT} make deploy
+        - docker tag ${IMAGE_BRANCH_COMMIT} ${GOOGLE_DOCKER_IMAGE}:latest
+        - IMAGE=${GOOGLE_DOCKER_IMAGE} make docker-push
```

What if we explicitly push the `latest` one, instead of all local `gcr.io/athenian-1/web-app` image tags?
```suggestion
        - IMAGE=${GOOGLE_DOCKER_IMAGE}:latest make docker-push
```